package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Dec 15 15:56:44 UTC 2020 - José Iván López González <jlopez@suse.com>
+
+- AutoYaST: added management of tmpfs file systems (related to
```

Comment:
NP: actually this is not only related to that jira entry. In fact, such jira is for adding tmpfs support in AutoYaST, although we took the opportunity for resurrecting it in the Partitioner too.